src/tools/kdb/cmerge.cpp

```diff
@@ -101,8 +101,9 @@ int CMergeCommand::execute (Cmdline const & cl ELEKTRA_UNUSED)
 		}
 		else
 		{
-			std::cerr << discard.size () << " keys exist in merge resultpath, will quit. Use -f to override the keys there."
+			std::cerr << "ERROR: " << discard.size () << " keys exist in resultpath. Use -f to override the keys there."
```

Comment:
simply throw exception here with this text, or print in the same format as other error messages (if you want specific return value it might be needed)